### PR TITLE
Updated plugin to be compatible with Python 3 and OpenSim 4.2.

### PR DESCRIPTION
**Pull request by tsalemink**

### CODE REVIEW

File: mapclientplugins/fieldworkgait2392geomstep/__init__.py [hunk 1]
```diff
@@ -1,12 +1,10 @@
-
-'''
+"""
 MAP Client Plugin
-'''
+"""
 __version__ = '0.1.2'
```
Comment (hsorby) [inline mapclientplugins/fieldworkgait2392geomstep/__init__.py]: Can you change this version to 0.2.0 please.

File: setup.py [hunk 2]
```diff
@@ -19,41 +19,49 @@
...
-    url='https://github.com/mapclient-plugins/fieldworkgait2392geomstep',
-    license='APACHE',
-    packages=find_packages(exclude=['ez_setup',]),
-    namespace_packages=['mapclientplugins'],
-    include_package_data=True,
-    package_data=package_data,
-    zip_safe=False,
-    install_requires=requires,
-    )
+      version='0.1.3',
```
Comment (hsorby) [inline setup.py]: Remind me to change this to scrape the version number from the plugin `__init__.py` file.

**Review by hsorby — CHANGES_REQUESTED** [review]

File: mapclientplugins/fieldworkgait2392geomstep/__init__.py (hunk 1 above)
Comment (tsalemink) [reply mapclientplugins/fieldworkgait2392geomstep/__init__.py]: Done.

File: setup.py (hunk 2 above)
Comment (tsalemink) [reply setup.py]: I have also made an attempt at this one. Seems to work.

**Pull request merged**